src/application/cli.py

```
@@ -0,0 +1,205 @@
+        
+        parameters = []
+        if args.parameters:
+            param_names = [p.strip() for p in args.parameters.split(",")]
+            for param in param_names:
+                try:
+                    parameters.append(ParameterType(param))
+                except ValueError:
+                    print(f"Error: Invalid parameter '{param}'")
+                    sys.exit(1)
+        
+        output_file = config.storage.base_path / f"{args.source}/processed/{args.country.lower()}_airquality_all_{datetime.now().strftime('%Y%m%d_%H%M%S')}.csv"
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)